internal/storager/storager.go

```diff
@@ -0,0 +1,6 @@
+package storager
+
+// Storager defines functions to interact with application storage service
+type Storager interface {
```

Comment:
A bit weird in English. How about:
```suggestion
type FileUploader interface {
```